lisa/tools/ip.py

```diff
@@ -74,6 +75,13 @@ def restart_device(self, nic_name: str) -> None:
             ),
         )
 
+    def setup_nic_at_boot(self, ip: str, nic_name: str) -> None:
```

Comment:
The method name is confusing. What's setup on the nic? And what does mean "at boot"?

Reply:
Maybe it doesn't need a new method, just add an argument `persistent` on original method.